package.json

```diff
@@ -11,7 +11,7 @@
     "lint:css": "stylelint 'src/**/*.{js,ts,tsx}'",
     "lint:js": "eslint ./ --ext js,ts,tsx --max-warnings 0",
     "lint:md": "markdownlint ./*.md {content,src}/**/*.{md,mdx}",
-    "prepare": "yarn build",
+    "prepare": "git submodule update --depth 1 --init && yarn build",
```

Comment:
This will load the most recent commit, with no history, during the `yarn install` process.